chart/templates/deployment.yaml

```
@@ -61,4 +66,12 @@ spec:
         - name: {{ $name }}
         {{- tpl (toYaml $config) $ | nindent 10 }}
         {{- end }}
+        {{- if $.Values.customNetworkConfig }}
+        - name: custom-network-config-volume
+          configMap:
+            name: custom-networks-config
```

Prefer a simpler name and need to templatize configMap name with previous recommendation.
```suggestion
        - name: config
          configMap:
            name: {{ include "hedera-explorer.fullname" . }}-config
```

---

configmap volume mounting is now moved to values.yaml